### PR TITLE
Support DEFAULT_LIMIT and DEFAULT_OFFSET

### PR DESCRIPTION
**Pull request by Seaven**
For improve code readability
* Replace -1 by `DEFAULT_LIMIT`
* Replace 0 by `DEFAULT_OFFSET`

### CODE REVIEW

File: fe/fe-core/src/main/java/com/starrocks/sql/optimizer/operator/logical/LogicalLimitOperator.java [hunk 1]
```diff
@@ -80,7 +80,7 @@ public int hashCode() {
 
     public static class Builder
             extends LogicalOperator.Builder<LogicalLimitOperator, LogicalLimitOperator.Builder> {
-        private long offset = -1;
+        private long offset = DEFAULT_OFFSET;
```
Comment (Astralidea) [inline fe/fe-core/src/main/java/com/starrocks/sql/optimizer/operator/logical/LogicalLimitOperator.java:84]: DEFAULT_OFFSET is 0 but here is previous value is -1, is that right?

File: fe/fe-core/src/main/java/com/starrocks/sql/optimizer/rule/transformation/MergeLimitWithLimitRule.java [hunk 2]
```diff
@@ -44,11 +45,11 @@ public MergeLimitWithLimitRule() {
 
         if (limit <= 0) {
             limit = 0;
-            offset = -1;
+            offset = Operator.DEFAULT_OFFSET;
         }
```
Comment (Astralidea) [inline fe/fe-core/src/main/java/com/starrocks/sql/optimizer/rule/transformation/MergeLimitWithLimitRule.java:49]: DEFAULT_OFFSET is 0 but here is previous value is -1, is that right?

File: fe/fe-core/src/main/java/com/starrocks/sql/optimizer/rule/transformation/MergeLimitWithLimitRule.java [hunk 3]
```diff
@@ -44,11 +45,11 @@ public MergeLimitWithLimitRule() {
...
 
         if (offset <= 0) {
-            offset = -1;
+            offset = Operator.DEFAULT_OFFSET;
         }
```
Comment (Astralidea) [inline fe/fe-core/src/main/java/com/starrocks/sql/optimizer/rule/transformation/MergeLimitWithLimitRule.java:53]: DEFAULT_OFFSET is 0 but here is previous value is -1, is that right?

**Review by kangkaisen — APPROVED** [review]

File: fe/fe-core/src/main/java/com/starrocks/sql/optimizer/operator/logical/LogicalLimitOperator.java (hunk 1 above)
Comment (Seaven) [reply fe/fe-core/src/main/java/com/starrocks/sql/optimizer/operator/logical/LogicalLimitOperator.java:84]: No problem~

**Review by HangyuanLiu — APPROVED** [review]

**Pull request merged**